tests/methods/aci-dsrg-mrpt2-4/input.dat

```diff
@@ -2,8 +2,10 @@ import forte
 
 memory 2 gb
 
-escf = -304.9057795504939463
-eaci = -305.39002965581
+#escf = -304.9057795504939463
+escf = -304.9057795492869332  # psi4@4dde4d8 after PR #1721
+#eaci = -305.39002965581
+eacidsrg = -305.390029654643570  # psi4@4dde4d8 after PR #1721
```

Comment:
This test case can be fixed by adding this to the psi4 input:
```
  ints_tolerance 0.0
```
Do you want to use this instead of changing the answers for all the failing test cases?

Reply:
Thank you for letting me know. I have add the keyword instead of changing the reference values.